python/cuml/test/dask/test_datasets.py

```diff
@@ -125,12 +125,16 @@ def test_make_regression(n_samples, n_features, n_informative,
         else:
             assert values.shape == (n_samples,), "values shape mismatch"
 
+        assert len(out.chunks[1]) == 1
```

Comment:
We should also assert that the number of chunks/partitions == `n_parts` here. As we continue to maintain this code, it will be very useful for us to know that the rechunk is not going to end up, for example, taking the number of partitions down to 1.